sdk/core/src/services/github/service.rs

```diff
@@ -114,11 +117,33 @@ impl GitHubService {
     }
 }
 
+#[async_trait]
+impl MongoService<Commit> for GitHubService {
+    fn store(&self) -> Arc<Store> {
+        self.store.clone()
+    }
+
+    /// Insert a document into a [Collection].
+    async fn insert<'a>(&self, doc: &mut Commit) -> Result<(), PlatformError> {
```

Comment:
Worth doing some validation here, since the ID gets set elsewhere?

Reply:
You bet, I'll get in there